packages/repository/test/unit/decorator/model-and-relation.decorator.unit.ts

```diff
@@ -93,14 +94,30 @@ describe('model decorator', () => {
     @property({type: 'string', id: true, generated: true})
     id: string;
 
-    @belongsTo(() => Customer)
+    @belongsTo(
+      () => Customer,
+      {},
+      {
+        id: true,
+        generated: false,
+      },
+    )
     customerId: string;
 
     // Validates that property no longer requires a parameter
     @property()
     isShipped: boolean;
   }
 
+  @model()
+  class RegistrationDate extends Entity {
+    @belongsToUniquely(() => Customer)
+    customerId: number;
+
+    @property()
+    registeredOn: Date;
+  }
+
```

Comment:
It's not so obvious that `RegistrationDate` deserves a `hasOne` relation from the `Customer`. Something like `A customer has a public profile` is probably better.